hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -882,6 +882,13 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+sw_test_timeout_ns=15_000_000"]
     }
+    {
+      name: chip_sw_entropy_src_csrng
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:entropy_src_csrng_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=15_000_000"]
```

Comment:
you can add some of the run time arguments here to speed up the test.
https://github.com/lowRISC/opentitan/blob/f771a932306664b63e473321040d27abf11343b8/hw/top_earlgrey/dv/chip_sim_cfg.hjson#L897

It helps quite a bit honestly.